tests/impls/vis_sal_classifier/test_rise_scoring.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+import pytest
+from smqtk_core.configuration import configuration_test_helper
+
+from xaitk_saliency.impls.vis_sal_classifier.rise_scoring import RISEScoring
+
+from tests import DATA_DIR, EXPECTED_MASKS_4x6
+
+
+class TestRiseScoring:
+
+    def test_configuration(self) -> None:
+        """
+        Test configuration aspects.
+        """
+        inst = RISEScoring(p1=44.777)
```

Comment:
Related to my other comment, but p1 I think should probably be bound between [0,1] as it should be the same as the RISE masking probability.